app/src/main/java/com/example/coronadiagnosticapp/ui/fragments/recorder/RecorderFragment.kt

```diff
@@ -143,6 +145,8 @@ class RecorderFragment : ScopedFragment() {
         //Setup Media Recorder for recording
         //Setup Media Recorder for recording
         mediaRecorder = MediaRecorder()
+        val visualizerView = visualizer;
+        val handler = Handler()
```

Comment:
Try to use coroutines instead of handler, if you find it difficult let me know :)